src/antd-components/DatePicker.tsx

```
@@ -0,0 +1,43 @@
+
+const onChange = (date, dateString) => {
+  console.log(date, dateString);
+};
+
+const defaultValue = [
+  dayjs("2000-01-01"),
+  dayjs("2000-01-03"),
+  dayjs("2000-01-05"),
+];
+
+const DatePicker = () => (
```

Biome

[error] Shouldn't redeclare 'DatePicker'. Consider to delete it or rename it.

'DatePicker' is defined here:


(lint/suspicious/noRedeclare)